entries/f/flaticon.com.json

```
@@ -3,7 +3,8 @@
     "domain": "flaticon.com",
     "contact": {
       "twitter": "flaticon",
-      "facebook": "flaticon"
+      "facebook": "flaticon",
+      "form": "https://www.flaticon.com/profile/support"
```

Email addresses are preferred over support forms. 
```suggestion
      "email": "support@flaticon.com"
```